### PR TITLE
Add Filter for wc_prices_include_tax() function return

### PR DESCRIPTION
**Pull request by sokolby**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
This PR allows use filter `wc_prices_include_tax` create custom logic for displaying prices based on some cases. 

### Other information:
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: includes/wc-conditional-functions.php [hunk 1]
```diff
@@ -361,7 +361,7 @@ function wc_shipping_enabled() {
 	 * @return bool
 	 */
 	function wc_prices_include_tax() {
-		return wc_tax_enabled() && 'yes' === get_option( 'woocommerce_prices_include_tax' );
+		return wc_tax_enabled() && apply_filters( 'wc_prices_include_tax', get_option( 'woocommerce_prices_include_tax' ) === 'yes' );
```
Comment (claudiosanches) [inline includes/wc-conditional-functions.php]: Need to fix the filter name, all filters and actions should start with `woocommerce_` and not `wc_`.

**Review by claudiosanches — CHANGES_REQUESTED** [review]

File: includes/wc-conditional-functions.php (hunk 1 above)
Comment (claudiosanches) [reply includes/wc-conditional-functions.php]: Also missing changelog in this PR.

Comment (sokolby) [reply includes/wc-conditional-functions.php]: done

**Review by claudiosanches — APPROVED** [review]
Thanks for your help!

**Pull request merged**